pkg/api/validation/validation.go

```diff
@@ -2189,6 +2189,9 @@ func ValidateService(service *api.Service) field.ErrorList {
 		allErrs = append(allErrs, field.Required(specPath.Child("ports"), ""))
 	}
 	if service.Spec.Type == api.ServiceTypeLoadBalancer {
+		if service.Spec.ClusterIP == "None" {
+			allErrs = append(allErrs, field.Invalid(specPath.Child("clusterIP"), service.Spec.ClusterIP, "invalid value when using load balancer service type"))
```

Comment:
can you also check for type == NodePort?  I opened 2 bugs, but it is the same problem.